src/ztc/api/tests/test_statustype.py

```diff
@@ -74,7 +73,12 @@ def test_get_detail(self):
             "volgnummer": statustype.statustypevolgnummer,
             "isEindstatus": True,
             "informeren": False,
-            "doorlooptijd": "P30D",
+            "doorlooptijdPeriode": 900,
+            "periodeEenheid": "werkdagen",
+            "doorlooptijd": {
+                "Periodeduur": 900,
+                "Periode-eenheid": statustype.periode_eenheid,
```

Comment:
Streepjes in attribuut namen zouden inprincipe niet gebruikt moeten worden (zie comment hierboven wat betreft naamgeving).